### ISSUE TITLE
Is there any way to insert a delay for lsp-auto-hover?

### ISSUE THREAD

**Issue by charlesprior** [kak-lsp/kak-lsp#426]
I would like to have lsp-auto-hover-enable'd, but I want a 1s or so delay so that the hover doesn't keep changing as I scroll through a file. Is this possible?

**Comment by clarfonthey** [comment]
Would also very much appreciate this. A 1s delay would seriously help when scrolling past a lot of symbols.

**Comment by krobelus** [comment]
Right, we don't have a good solution here yet. I'm not sure what's the best approach here. Maybe roll your own `lsp-auto-hover` by running this in another shell:

    while sleep 1; do echo eval -client client0 lsp-hover | kak -p my-session; done

The UI is not optimal.. this might be a nice use case for https://github.com/mawww/kakoune/issues/4127
Though this works now, and you can wrap it in a Kakoune command:

```kak
define-command lsp-auto-hover-in-client %{
    nop %sh{
        (
            while echo eval -client $kak_client lsp-hover | kak -p $kak_session
            do
                sleep 1
            done
        ) >/dev/null 2>&1 </dev/null &
    }
}
```

you could also try something like `set global idle_timeout 500` but that will also slow down other hooks (see also https://github.com/kak-lsp/kak-lsp/issues/394 and https://github.com/kak-lsp/kak-lsp/issues/429)

**Comment by clarfonthey** [comment]
Hmm, that's not quite what I was looking for but I'll try it. Right now the rapid flashing from the hover sometimes make things so unbearable I have to disable it.

**Comment by krobelus** [comment]
BTW #570 allows to show hover info in a separate buffer, maybe that works better? I'll merge that soon
